### ISSUE TITLE
ARIA issues with Modal

### ISSUE THREAD

**Issue by nstuyvesant** [bestguy/sveltestrap#398]
Looking at the rendered HTML for the Modal, seeing an attribute called "arialabelledby" instead of "aria-labelledby". Also, the value is empty ("") but should be the HTML id of the element with class="modal-title". Perhaps you could internally generate an id for it so it's unique?

**Comment by bestguy** [comment]
Released in [v5.7.0](https://github.com/bestguy/sveltestrap/releases/tag/v5.7.0) thanks for the heads up 👍
